recipe/meta.yaml

```diff
@@ -16,25 +16,27 @@ source:
     #                                                        #
     # ref: https://bugs.exim.org/show_bug.cgi?id=2157        #
     ##########################################################
-    - bug_2157.patch
+    # - bug_2157.patch
 
 build:
-  number: 1002
+  number: 1000
```

Comment:
Will clobber be unhappy about this? I think this was upped to 1002 because the current build number is 2

Reply:
Never mind, I just saw the new version number.

Reply:
version number changed